MedicalDevices/README.txt

```diff
@@ -4,44 +4,51 @@
 
 Concept
 -------
-
```

Comment:
We trying to transition all documentation to markdown files across GitHub. Could you reformat all the README files in the repository to markdown?